README.md

```diff
@@ -49,6 +49,48 @@ If you'd like to protect your Sentry install with SSL/TLS, there are
 fantastic SSL/TLS proxies like [HAProxy](http://www.haproxy.org/)
 and [Nginx](http://nginx.org/). You'll likely want to add this service to your `docker-compose.yml` file.
 
+OR do the following things to enable auto redirect to https behind a reverse proxy
+
+Uncomment the following in `docker-compose.yml` nginx directive
+```
+ #expose:
+ #  - '443'
+ ports:
+ #  - '443:443'
+   - '$SENTRY_ONPREMISE_PORT:80/tcp'
+```
+
+And uncomment the following in your sentry.config.py
```

Comment:
I think instead of repeating the same config here in the README, we can mark the start and end of this section clearly with `SSL CONFIG START`, `SSL_CONFIG_END` and just say "Uncomment all the config inside the `SSL CONFIG` section.

Even better, we may wrap that section in an
```py
if env('SENTRY_USE_SSL'):
   ...
```

block, and just tell people to enable this in their `.env` file.

Reply:
i will confirm this as working and change in default sentry.conf, and update readme

Reply:
confirmed and updated